### ISSUE TITLE
Extend the default base controller of Laravel

### ISSUE THREAD

**Issue by thewebartisan7** [regnerisch/laravel-beyond#63]
Instead of adding for each controller the default Laravel traits for controller, would be better to keep the default Laravel controller as Base controller and then extend it.

So the controller.stub would be:

```php
<?php

namespace App\{{ namespace }}\Controllers;

use Support\Packages\Laravel\Controllers\Controller;

class {{ className }} extends Controller
{
    // Methods here...
}
```

And we can add the base controller for example inside Support/Packages/Laravel/Controller with default content:

```php
<?php

namespace Support\Packages\Laravel\Controllers;

use Illuminate\Foundation\Auth\Access\AuthorizesRequests;
use Illuminate\Foundation\Bus\DispatchesJobs;
use Illuminate\Foundation\Validation\ValidatesRequests;
use Illuminate\Routing\Controller as BaseController;

class Controller extends BaseController
{
    use AuthorizesRequests, DispatchesJobs, ValidatesRequests;
}
```

**Comment by regnerisch** [comment]
The big question is why should we change this? If you can give me a good reason we may change this in a future release.

**Comment by regnerisch** [comment]
I will revert this behavior in v4. We had a base controller before v3. I see your points.

**Comment by regnerisch** [comment]
Resolved in #71
